Source/UINavigation/Private/UINavWidget.cpp

```diff
@@ -801,11 +801,13 @@ void UUINavWidget::RemoveFromParent()
 	Super::RemoveFromParent();
 }
 
+#if ENGINE_MAJOR_VERSION == 5 && ENGINE_MINOR_VERSION < 1
```

Comment:
What happened here? Is this no longer available in 5.1?

Reply:
Yes for some reason the "OnLevelRemovedFromWorld" isn't on the UWidget anymore

The other solution would be to remove the "virtual" from the function in the .h, but that would simply means the function still exist but is never called by the engine.

Reply:
I'd say just remove the function from UINavWidget entirely.
I don't even know what it's doing here in the first place. (I'm not the one who put this here)

Reply:
Done